doc/appInstance.md

```diff
@@ -0,0 +1,31 @@
+# App instance configuration
+
+## Edit configuration files
+The file [src/config/AppInstance.js](../src/config/AppInstance.js) defines constants specific to your application
+instance: you will need to change these values when creating a new staging or prod version of your application.
+
+* **AZURE_TENANT_ID** is the identifier of the Azure tenant your application is registered in.\
+  You can find it in the Azure portal, in the *App registration* blade, in the *Overview* menu.
+* **APP_REGISTRATION_CLIENT_ID** is the identifier of your app registration.\
+  You can find it in the Azure portal, in the *App registration* blade, in the *Overview* menu.
```

Comment:
Same

Reply:
Replaced by `Azure portal > App registration > Overview` for clarity. Thank you :+1: